tests/www/v1/stock2shop/dal/channels/os/data/HelperTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace tests\v1\stock2shop\dal\channels\os\data;
+
+use stock2shop\dal\channels\os;
+use tests;
+
+/**
+ * Helper Test
+ *
+ * Example unit test
+ *
+ * All unit test classes must extend the tests\TestCase base class.
+ */
+class HelperTest extends tests\TestCase
+{
+
+    /**
+     * Example unit test
+     *
+     * Note the naming convention used for testing a function.
+     * e.g. test[FunctionName]()
+     *
+     * The path for the test class must be  similar to the path for the test function.
```

Comment:
rather put the generic comments about how unit tests work in readme and you can link to this file as an example

Reply:
Added section to the main README.md file of the repository under "Unit Tests".